garrysmod/gamemodes/terrortown/gamemode/cl_targetid.lua

```diff
@@ -34,25 +34,25 @@ end
 local indicator_mat = Material("vgui/ttt/sprite_traitor")
 local indicator_col = Color(255, 255, 255, 130)
 
-local client, plys, ply, pos, dir, tgt
-local GetPlayers = player.GetAll
+local client, inext, plys, pos, dir, tgt
+local num = 0
+local GetPlayers = player.Iterator
 
 local propspec_outline = Material("models/props_combine/portalball001_sheet")
 
 -- using this hook instead of pre/postplayerdraw because playerdraw seems to
 -- happen before certain entities are drawn, which then clip over the sprite
 function GM:PostDrawTranslucentRenderables()
    client = LocalPlayer()
-   plys = GetPlayers()
+   inext, plys = GetPlayers()

```

Comment:
This looks like a mess. IMO it should either be left as is, or the loops be converted to use `player.Iterator`. I can't imagine caching the 3 returns is going to save a lot of performance, the most important performance savings are already done by `player.Iterator`, by caching the player table.

Reply:
Does this apply to the ones in [init.lua](https://github.com/Facepunch/garrysmod/pull/2062/files#diff-7a8b22d2b219d4f4dd9b2d78f1ef16e700592dc75bd8d9ecf42353db31ab8758R530) and [traitor_state.lua](https://github.com/Facepunch/garrysmod/pull/2062/files#diff-5da3e5f509939356952f4e4a75a4f1def4798c2439bc831223d9b446e8fff24eR95) as well?

Reply:
Yeah